src/lib/components/Tabs/index.js

```diff
@@ -64,6 +71,27 @@ export class Tabs extends BaseElement {
     `;
   }
 
+  loadmoreTab() {
```

Comment:
Nit, casing: `loadMoreTab`, also applies to `loadMorePanel` and `onClickLoadMore`.